backend/battle/tests/test_view.py

```diff
@@ -0,0 +1,123 @@
+from django.test import TestCase, Client
+from model_bakery import baker
+from django.urls import reverse
+from battle.models import Team
+from users.models import User
+
+
+class AssociatePokemonsToBattleTest(TestCase):
```

Comment:
Is a good practice to put the name of the test the same of view plus `Test`

Reply:
hmm great!